docs/HANDLED_RULES.md

```diff
@@ -343,6 +344,27 @@ class SelfAssignement {
 
 -----
 
+#### "toString()" and "clone()" methods should not return null ([Sonar Rule 2225](https://rules.sonarsource.com/java/type/Bug/RSPEC-2225))
+
+For the return statements inside "toString()", this processor replaces the return expression with an empty string. 
```

Comment:
I think you should explicitly state here that the processor is incomplete and ignores `clone()` methods. Ideally, we'd pull that information from the source code annotation, but for now we just have to duplicate the information.